pnda_env_example.yaml

```diff
@@ -95,6 +95,9 @@ openstack_parameters:
   # Address of gateway for Kakfa interface on public producer subnet
   publicProducerSubnetGateway: 10.0.2.1
 
+  #Name server -  a comma separated address
+  dns_nameservers: "172.23.136.144,8.8.8.8"
```

Comment:
What do we do with these here, given our use of Consul now - and why is this OpenStack specific?

Reply:
In a typical Openstack enterprise setup, DNS service provided by Openstack resolve the names within the Openstack cluster. But for Openstack nodes that need to resolve names  for ex google.com, ntp.server.ubuntu.com nodes need to have corporate separate DNS server or a public DNS server. Hence this is specific to Openstack. For the same reason, the existing heat-template repo has the same